spirv_cross_c.h

```diff
@@ -257,15 +257,25 @@ typedef enum spvc_msl_platform
 	SPVC_MSL_PLATFORM_MAX_INT = 0x7fffffff
 } spvc_msl_platform;
 
+/* Maps to C++ API. */
+typedef enum spvc_msl_index_type
+{
+	SPVC_MSL_INDEX_TYPE_NONE = 0,
+	SPVC_MSL_INDEX_TYPE_UINT16 = 1,
+	SPVC_MSL_INDEX_TYPE_UINT32 = 2,
+	SPVC_MSL_INDEX_TYPE_MAX_INT = 0x7fffffff
+} spvc_msl_index_type;
+
 /* Maps to C++ API. */
 typedef enum spvc_msl_vertex_format
 {
 	SPVC_MSL_VERTEX_FORMAT_OTHER = 0,
 	SPVC_MSL_VERTEX_FORMAT_UINT8 = 1,
-	SPVC_MSL_VERTEX_FORMAT_UINT16 = 2
+	SPVC_MSL_VERTEX_FORMAT_UINT16 = 2,
+	SPVC_MSL_VERTEX_FORMAT_UINT32 = 3,
 } spvc_msl_vertex_format;
 
-/* Maps to C++ API. */
+/* Maps to C++ API. Deprecated; use spvc_msl_vertex_attribute2. */
```

Comment:
It doesn't feel right to deprecate this if vecsize is only relevant for a very specific tessellation control input implementation.

Would it make sense to add a separate entry point which just adds vecsize information ala
`spvc_msl_vertex_attribute_explicit_vecsize(comp, location, vecsize)` or something like that?

Reply:
This has already been replaced with `spvc_msl_shader_input`.